dev-python/astroid/astroid-2.1.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Abstract Syntax Tree for logilab packages"
+HOMEPAGE="https://github.com/PyCQA/astroid https://pypi.org/project/astroid/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~amd64-linux ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x64-macos ~x86 ~x86-linux ~x86-macos"
+IUSE="test"
+
+# Version specified in __pkginfo__.py.
+RDEPEND="
+	dev-python/lazy-object-proxy[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	dev-python/wrapt[${PYTHON_USEDEP}]
+	$(python_gen_cond_dep '<=dev-python/typed-ast-1.2.0[${PYTHON_USEDEP}]' python3_4 python3_5 python3_6 pypy3)
+	virtual/python-typing[${PYTHON_USEDEP}]"
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? (
+		${RDEPEND}
+		dev-python/nose[${PYTHON_USEDEP}]
+		$(python_gen_cond_dep 'dev-python/numpy[${PYTHON_USEDEP}]' 'python*')
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/python-dateutil[${PYTHON_USEDEP}]
+	)"
+
+python_prepare_all() {
+	# Disable failing tests
+
+	# no idea why this test fails
+	sed -i -e "s/test_knownValues_get_builtin_module_part/_&/" \
+		astroid/tests/unittest_modutils.py || die
+
+	distutils-r1_python_prepare_all
+}
+
+python_test() {
+	${EPYTHON} -m pytest --pyarg astroid/tests || die "tests failed"
```

Comment:
We generally want to run tests in verbose mode, for more insightful logging in bug reports.